rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/asset_search/action.py

```diff
@@ -22,6 +22,7 @@ def run(self, params={}):
             parameters.append(("sort", f"{key},{value}"))
 
         if size > 500:
+            self.connection.logger.info(f"'{size}' too large, set to max size of 500.")
```

Comment:
```suggestion
            self.logger.info(f"'{size}' too large, set to max size of 500.")
```
Actions and triggers also have logger instances, no need to go to the connection for it :)